Applications/Utils/FileConverter/convertGEO.cpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include <string>
+#include <vector>
+
+#include <tclap/CmdLine.h>
+
+#include "Applications/ApplicationsLib/LogogSetup.h"
+
+#include "GeoLib/IO/readGeometryFromFile.h"
+#include "GeoLib/IO/writeGeometryToFile.h"
+#include "GeoLib/GEOObjects.h"
+
+
+int main (int argc, char* argv[])
+{
+    ApplicationsLib::LogogSetup logog_setup;
+
+    TCLAP::CmdLine cmd("Converts OGS geometric file into another file format.", ' ', "0.1");
```

Comment:
It could be that an user doesn't know what *another file format* means. Maybe in the description of the tool the concrete possiblities: (1) gml to gli, and (2) gli to gml can be  documented . Of course one could convert (3) gml to gml and (4) gli to gli also. But  conversion (3) and (4) seems pretty useless.